lang/main.json

```diff
@@ -393,8 +393,7 @@
         "toggleFilmstrip": "Show or hide video thumbnails",
         "toggleScreensharing": "Switch between camera and screen sharing",
         "toggleShortcuts": "Show or hide keyboard shortcuts",
-        "videoMute": "Start or stop your camera",
-        "videoQuality": "Manage call quality"
```

Comment:
This doesn't seem related to the PR. Just wanted to check if you have accidentally committed it?

Reply:
My bad! Just saw it is in separate commit so I guess it is not accidental :)